app/View/Edit.js

```diff
@@ -1,21 +1,102 @@
 import React from 'react';
-import ViewActions from '../Component/ViewActions';
+import Inflector from 'inflected';
+import { shouldComponentUpdate } from 'react-immutable-render-mixin';
 import Compile from '../Component/Compile';
 
+import ViewActions from '../Component/ViewActions';
+import EditActions from '../Actions/EditActions';
+import EditStore from '../Stores/EditStore';
+import FormFields from '../Component/Form/FormFields';
+
 class EditView extends React.Component {
+    constructor() {
+        super();
+
+        this.state = EditStore.getState();
+        this.shouldComponentUpdate = shouldComponentUpdate.bind(this);
+        this.save = this.save.bind(this);
```

Comment:
not a big fan of this, please bind the function when you pass it, not in the constructor.